charts/alfresco-repository/tests/ingress_test.yaml

```diff
@@ -19,28 +19,6 @@ tests:
         template: ingress.yaml
```

Comment:
you can remove all the template inside each test since this suite include only one template